features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin_new/source/src/app/components/AdminPages/Addons/InputListBase.jsx

```diff
@@ -73,8 +79,11 @@ const InputList = (props) => {
                                     onChange={handleInput}
                                     label={inputLabelPrefix + ' ' + labelCounter++}
                                     value={item.value}
-                                    helperText={helperText}
+                                    helperText={
+                                        userInputItems.filter((obj) => obj.key === item.key)[0].error || helperText
```

Comment:
Let's do a find here, Filter will iterate over all the values in the array where find will break in the fist finding , and won't need to use `[0]`

Reply:
Fixed with commit: https://github.com/wso2/carbon-apimgt/pull/8535/commits/5ea6a84b8d1a47454611ce6d7f6cd7bac359513f